spec/request/in_progress_forms_request_spec.rb

```diff
@@ -18,7 +18,22 @@
     )
   end
 
-  describe 'GET' do
+  describe '#index' do
```

Comment:
So, uh, forgive my ignorance, but I don't speak Ruby, I'm afraid. How do `'#index'`, `'#show'`, and `'#update'` correspond to the HTTP verbs?